src/vm/gcenv.ee.cpp

```diff
@@ -1176,13 +1176,13 @@ void GCToEEInterface::DiagWalkFReachableObjects(void* gcContext)
 // don't get confused.
 void WalkMovedReferences(uint8_t* begin, uint8_t* end, 
                          ptrdiff_t reloc,
-                         size_t context, 
-                         BOOL fCompacting,
-                         BOOL fBGC)
+                         void* context, 
+                         bool fCompacting,
+                         bool fBGC)
```

Comment:
I figured that these function pointer types are exposed via the interface by virtue of being arguments to interface functions. I can change them back to `BOOL` or fix the call sites - both are fine with me, whatever you'd prefer.